images/sds-local-volume-scheduler-extender/src/pkg/cache/cache.go

```diff
@@ -43,18 +47,56 @@ type pvcCache struct {
 }
 
 // NewCache initialize new cache.
-func NewCache(logger logger.Logger) *Cache {
-	return &Cache{
-		log: logger,
+func NewCache(logger logger.Logger, pvcExpDurSec time.Duration) *Cache {
```

Comment:
> var pvcExpDurSec is of type time.Duration; don't use unit-specific suffix "Sec" (ST1011)go-staticcheck

This warning makes sense, because it's no longer bound to any interval like "seconds" after being converted to `time.Duration`